ivy/container/container.py

```diff
@@ -150,9 +150,10 @@ def __add__(self, other):
                           [8.1, 9.3, 3.4]])
         }
         """
-        return ivy.Container.multi_map(
-            lambda xs, _: operator.add(xs[0], xs[1]), [self, other], map_nests=True
-        )
+        # return ivy.Container.multi_map(
```

Comment:
actually, `__add__` should not call `static_add`, as `__add__` should also support adding non-array types, such as `int` etc.